api/application.go

```diff
@@ -140,8 +147,8 @@ func (h ApplicationHandler) Get(ctx *gin.Context) {
 	r := Application{}
 	r.With(m, tags)
 	r.WithArchetypes(archetypes)
-	r.WithSourcedTags(archetypeTags, "archetype")
-	r.WithSourcedTags(resolver.AssessmentTags(), "assessment")
+	r.WithVirtualTags(archetypeTags, "archetype")
+	r.WithVirtualTags(resolver.AssessmentTags(), "assessment")
```

Comment:
Did you mean to use new the _const_ here?

Reply:
I did, thanks for pointing that out.